src/index.html

```diff
@@ -7,10 +7,35 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <title>Search bar airbnb</title>
   </head>
   <body>
     <h1>Search bar airbnb</h1>
+    <form 
+    action="/" 
+    method="post"
+    data-qa="big">
+      <label>
+        <input
+        data-qa="keypress"
+        type="search" 
+        name="search" 
+        class="largeInput"
+        placeholder="Try &quot;Los Angeles&quot;">
```

Comment:
```suggestion
        <input
          data-qa="keypress"
          type="search" 
          name="search" 
          class="largeInput"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```
Давай скрізь пофіксимо форматування